analyses/telomerase-activity-prediction/02-Comparing-Counts-versus-FPKM.R

```diff
@@ -0,0 +1,77 @@
+library('pracma')
+library('ggplot2')
+library('gplots')
+library('dplyr')
+library('ggpubr')
+stringsAsFactors=FALSE
+library(circlize)
+library(stringr)
+library(gridBase)
+library(gridGraphics)
+
+
+
+
+################################################ Comparing Counts versus FPKM (Figure 1) ############################################################################################
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git")) 
+
+A = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAPolya_counts.txt")   ### Variable representing Telomerase activity-prediction for Polya_counts
+				 
+B = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAStranded_counts.txt")  ### Variable representing Telomerase activity-prediction for Stranded_counts
+				 
+C = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAPolya_FPKM.txt")      ### Variable representing Telomerase activity-prediction for PolyA_FPKM data
+				 
+D = file.path(root_dir, "analyses", "telomerase-activity-prediction",
+                 "results", "TelomeraseScores_PTBAStranded_FPKM.txt")   ### Variable representing Telomerase activity-prediction for Stranded_FPKM data
+
+PTBA_GE_PolyA_TMScores1 = read.table(A,sep='\t',head=T)
+colnames(PTBA_GE_PolyA_TMScores1)[colnames(PTBA_GE_PolyA_TMScores1)=="NormEXTENDScores"]='NormEXTENDScores_PolyACounts'
+
+PTBA_GE_Standard_TMScores1 = read.table(B,sep='\t',head=T)
+colnames(PTBA_GE_Standard_TMScores1)[colnames(PTBA_GE_Standard_TMScores1)=="NormEXTENDScores"]='NormEXTENDScores_StrandedCounts'
+
+PTBA_GE_PolyA_TMScores2 = read.table(C,sep='\t',head=T)
+colnames(PTBA_GE_PolyA_TMScores2)[colnames(PTBA_GE_PolyA_TMScores2)=="NormEXTENDScores"]='NormEXTENDScores_PolyA_FPKM'
+
+PTBA_GE_Standard_TMScores2 = read.table(D,sep='\t',head=T)
+colnames(PTBA_GE_Standard_TMScores2)[colnames(PTBA_GE_Standard_TMScores2)=="NormEXTENDScores"]='NormEXTENDScores_Stranded_FPKM'
+
+PTBA_GE_PolyA_TMScores = merge(PTBA_GE_PolyA_TMScores1,PTBA_GE_PolyA_TMScores2,by='SampleID')
+PTBA_GE_Standard_TMScores = merge(PTBA_GE_Standard_TMScores1,PTBA_GE_Standard_TMScores2,by='SampleID')
+
+
+pdf('plots/PTBA_GE_Score_AllScatter.pdf')
```

Comment:
I would recommend creating a variable at the top of the script with this file name, OR (and perhaps a better solution?), pass in the output plot name as a command line argument into the script, which you can integrate into the shell script.